src/main/webapp/content/scss/_data-table-icons.scss

```diff
@@ -22,6 +22,10 @@
     content: '⟩';
 }
 
+.datatable-icon-down:before {
```

Comment:
https://stylelint.io/user-guide/rules/list/selector-pseudo-element-colon-notation/

Reply:
Also for consistency you could also adapat this in the rest of the file.